packages/lit-dev-content/site/css/fonts/open-sans.css

```diff
@@ -5,7 +5,7 @@
   font-style: normal;
   font-weight: 300;
   font-display: swap;
-  src: url(https://fonts.gstatic.com/s/opensans/v18/mem5YaGs126MiZpBA-UN_r8OUuhp.woff2) format('woff2');
+  src: url(/fonts/opensans/OpenSans-300.woff2) format('woff2');
```

Comment:
should the filename have some indication that these are for latin chars only?

Reply:
There wasn't really an indication before and we don't even know how we are going to handle localization. I say we hold off on this for now unless you think it's something we should push for?

Reply:
I'm guessing the original gstatic url came from something like this https://fonts.googleapis.com/css2?family=Roboto+Mono based on src and unicode-range. If the woff2 files being added here were downloaded by following the src url, I assume it contains only latin characters of the font and could potentially be misleading down the line if we need more than latin chars.

Having said that, the `unicode-range` is still present here which does indicate this is a subset, and we don't have plans for additional localization anytime soon. By then we may have also moved forward with the font update situation making this moot. So not a big deal if we don't want to make any changes on this.

Reply:
I'll include it. I just can't read mangled urls quite as good as you 😄

Reply:
I thought everyone could encode/decode all forms of mangling in their head 🤖